tests/test_calendar.py

```
@@ -24,335 +24,331 @@
+
+    # day_from_ymd()
+    assert day_from_ymd(2000,1,1) == 0
+    assert type(day_from_ymd(2000,1,1)) is int
+    assert day_from_ymd(2000,1,1.) == 0.
+    assert type(day_from_ymd(2000,1,1.)) is float
+
+    assert day_from_ymd(2000,2,[27,28,29]).tolist() == [57,58,59]
+    assert day_from_ymd(2000,[1,2,3],1).tolist() == [ 0,31,60]
+    assert day_from_ymd([2000,2001,2002],1,1).tolist() == [0,366,731]
+
+    with pytest.raises(JVF): day_from_ymd(2000, 1,  0, validate=True)
```

Ruff (0.15.6)

[error] Multiple statements on one line (colon)

(E701)